extensions/wikia/ARecoveryEngine/js/adBlockRecovery.js

```diff
@@ -42,13 +42,20 @@ define('ext.wikia.aRecoveryEngine.adBlockRecovery', [
 
 	function isEnabled() {
 		var context = adContext.getContext(),
-			enabled = adBlockDetection.isEnabled() &&
-				(context.opts.sourcePointRecovery || context.opts.pageFairRecovery);
+			enabled = context.opts.sourcePointRecovery || context.opts.pageFairRecovery;
 
 		log(['isEnabled', enabled], log.levels.debug, logGroup);
 		return enabled;
 	}
 
+	function isSourcePointEnabled() {
```

Comment:
why not to use `isEnabled` from `'ext.wikia.aRecoveryEngine.sourcePoint.recovery'` ?